exercises/ros/services/README.md

```diff
@@ -0,0 +1,36 @@
+# Foundation Course: Service / Client
```

Comment:
Move this to the patterns.md file instead. You can add a second level header (##) for Service client.